tests/unit/data_formats/test_yolo_format.py

```diff
@@ -1118,8 +1314,20 @@ def test_can_report_missing_subset_folder(self, test_dir):
         with pytest.raises(InvalidAnnotationError, match="subset image folder"):
             Dataset.import_from(dataset_path, self.IMPORTER.NAME).init_cache()
 
+    def test_can_report_missing_ann_file(self, test_dir):
+        # YOLOv8 does not require annotation files
+        pass
```

Comment:
I don't think an empty test is needed here, especially since the following test covers the same scenario.

Reply:
I made an empty test so that a test with same name from the parent class would not be run.
I would not say the scenario is the same. `test_can_report_missing_ann_file` check that the importer fails, `test_can_import_with_missing_ann_file` checks that the importer imports successfully.

Reply:
> I made an empty test so that a test with same name from the parent class would not be run.

Ah, okay. In that case, could you add a comment explaining that?

Reply:
added a comment